<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutocompletePage</class>
 <widget class="QWidget" name="AutocompletePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QCheckBox" name="useDocumentChk">
     <property name="text">
      <string>Use current document</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useApiChk">
     <property name="text">
      <string>Use API files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="replaceWordChk">
     <property name="text">
      <string>Replace word</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="matchCaseChk">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="thresholdSpin">
       <property name="specialValueText">
        <string/>
       </property>
       <property name="suffix">
        <string> symbol(s)</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>181</width>
       <height>201</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
